NameLayerMain/src/vg/civcraft/mc/namelayer/group/Group.java

```diff
@@ -317,23 +318,38 @@ public PlayerType getCurrentRank(UUID uuid) {
 	 * @param type- The PlayerType to add. If a preexisting PlayerType is found, 
 	 * it will be overwritten.
 	 */
-	public void addMember(UUID uuid, PlayerType type) {
+	
+	public void addMember(UUID uuid, PlayerType type){
+		addMember(uuid,type,true);
```

Comment:
Yes this change is intentional. All previous callers are expecting a db change to happen. So I kept the calling method the same but added a bool on the end. These allowed me in the mercury messaging to have the call similar but explicitly only change the cache.